src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -24,74 +27,63 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class Locomotive implements Conductor<Locomotive> {
+@Listeners({TestListener.class, WebDriverListener.class})
+public class Locomotive extends Watchman implements Conductor<Locomotive> {
+
+    //Declare ThreadLocal Driver (ThreadLocalMap) for ThreadSafe Tests
+    private static ThreadLocal<WebDriver> driver = new ThreadLocal<>();
 
     public ConductorConfig configuration;
-    public WebDriver driver;
```

Comment:
I think changing the driver access to a getter will break a lot of projects but is a really good change. I think we should make clear in the release notes that the public access got removed and might require some code adjustments.